src/test/scala/treadle/executable/SymbolTableSpec.scala

```diff
@@ -16,15 +16,18 @@ limitations under the License.
 
 package treadle.executable
 
+import java.io.{ByteArrayOutputStream, PrintStream}
+
 import firrtl.graph.CyclicException
 import firrtl.stage.FirrtlSourceAnnotation
 import firrtl.transforms.DontCheckCombLoopsAnnotation
+import logger.{LazyLogging, LogLevel, Logger}
 import treadle._
 import org.scalatest.freespec.AnyFreeSpec
 import org.scalatest.matchers.should.Matchers
 
 //scalastyle:off magic.number
-class SymbolTableSpec extends AnyFreeSpec with Matchers {
+class SymbolTableSpec extends AnyFreeSpec with Matchers with LazyLogging {
```

Comment:
Should the `LazyLogging` changes be a separate PR?

Reply:
Done